python/lsst/fgcmcal/fgcmBuildStars.py

```diff
@@ -160,15 +160,11 @@ class FgcmBuildStarsConfig(pexConfig.Config):
         default="calib_psf_candidate"
     )
     doSubtractLocalBackground = pexConfig.Field(
-        doc="Subtract the local background before performing calibration?",
+        doc=("Subtract the local background before performing calibration? "
+             "This is only supported for circular aperture calibration fluxes."),
         dtype=bool,
         default=False
     )
-    calibFluxApertureRadius = pexConfig.Field(
-        doc="Calibration flux aperture radius (pixels)",
-        dtype=float,
-        default=12.0
-    )
```

Comment:
I'm not a fan of adding & removing in a single ticket... 😉

Reply:
This is outdated because of a rebase.  So it's like it never happened!